src/sql/azdata.d.ts

```diff
@@ -5064,8 +5059,8 @@ declare module 'azdata' {
 		}
 		export interface IKernelSpec {
 			name: string;
+			display_name: string;
```

Comment:
Going from optional to required is a breaking change so we shouldn't be doing that - revert this for now and then we'll have to take a closer look at whether we can work around that somehow